packages/components/src/components/header/header.styles.ts

```diff
@@ -39,11 +39,11 @@ export default css`
    */
   .header__logo ::slotted(a),
   .header__logo ::slotted(a:hover) {
-    color: var(--syn-color-primary-600);
+    color: var(--syn-logo-color);
   }
 
   .header__logo ::slotted(a:is(:focus-visible)) {
-    outline: var(--syn-focus-ring);
+    outline: var(--syn-logo-color);
```

Comment:
we need the fallback token here


```suggestion
    outline: var(--syn-logo-color, var(--syn-focus-ring));
```